pkg/webui/components/navigation/side/item/__snapshots__/index_test.js.snap

```diff
@@ -23,7 +23,7 @@ exports[`SideNavigationItem is collapsable should match snapshot 1`] = `
       ]
     }
     onClick={[Function]}
-    onExpand={[Function]}
+    onExpand={[MockFunction]}
```

Comment:
what happened here?

Reply:
I needed to update jest to make it work with babel 7, which seemed to have changed some stuff in snapshot creation. I updated the snapshots to make the tests pass again.